profile-datastore-common-test/src/main/java/com/telecominfraproject/wlan/profile/datastore/BaseProfileDatastoreTest.java

```diff
@@ -291,10 +313,9 @@ public void testProfilePagination()
     
     @Test
     public void testChildProfiles() {
-    	int nextId = (int) testSequence.getAndIncrement();
 
-    	Profile profile_c1 = createProfileObject(nextId);
-    	Profile profile_c2 = createProfileObject(nextId);
+    	Profile profile_c1 = createProfileObject(getNextCustomerId());
+    	Profile profile_c2 = createProfileObject(getNextCustomerId());
```

Comment:
Nope, by changing code to use different customerId per profile you have changed the spirit of the test. These profiles are meant to be owned by the same customer (they are children of profile_p1). To do it right the createProfileObject has to be augmented with profileName (and/or type) arguments, and that should be used to make it work with the new constraint on the name/type.